src/extreme_filter.jl

```diff
@@ -49,9 +49,9 @@ julia> extreme_filter(max, M; dims=(1, )) # max-filter along the first dimension
 connectivity, or a `AbstractArray{<:CartesianIndex}` array with each element indicating the
 displacement offset to its center element.
 
-```
-julia> Ω_mask = Bool[1 1 0; 1 1 0; 1 0 0] # custom neighborhood in mask format
-3×3 Matrix{Bool}:
+```jldoctest extreme_filter
+julia> Ω_mask = centered(Bool[1 1 0; 1 1 0; 1 0 0]) # custom neighborhood in mask format
+3×3 OffsetArray(::Matrix{Bool}, -1:1, -1:1) with eltype Bool with indices -1:1×-1:1:
  1  1  0
```

Comment:
Warning! be aware that in case of non symmetrical SE, the SE should be transposed before exteme_filter call
Its the difference between erode/dilate and minkowski -,+
see https://people.cmm.minesparis.psl.eu/users/serra/cours/pdf/en/ch2en.pdf p7 or soille book
Anyway transpose is ship so may we could always run that transform instead of detect non symmetric property first

Reply:
I see; this seems to be a comment on the conceptual confusion, that is, dilation is a Minkowski addition iff the SE is symmetric with respect to its origin.

This can be part of the documentation, but for `extreme_filter`, and `dilate`, `erode`. I believe we don't need to change it.